pegasus/sites.v3/code.org/public/dance.haml

```diff
@@ -61,6 +61,12 @@ theme: responsive_full_width
     = view :"dance/dance_inspirational_videos"
 
 %section
+  .wrapper
+    #behind-the-scenes
```

Comment:
Is the intention for this div to be used as an anchor link?

Reply:
Yes.  Is there a better way to do it?

Reply:
Yeah! You can add it to the `%section` tag above since that's a semantic element that already exists, plus it'll add some nice padding above the heading.

Reply:
Done; thanks!